gen/gen.go

```diff
@@ -69,14 +71,26 @@ package docs
 
 import (
 	"github.com/swaggo/swag"
+	"os"
+	"io/ioutil"
 )
 
 var doc = {{.Doc}}
 
 type s struct{}
 
 func (s *s) ReadDoc() string {
-	return doc
+	file, err := os.Open("{{.SwaggerJSON}}")
```

Comment:
@wksw would you tell us why there is using `{{.SwaggerJSON}}`? It seems like will go wrong .

Reply:
use `{{.SwaggerJSON}}` it  means read from file not from variable `doc`(line 78)
the variable doc's content is same with the file `{{.SwaggerJSON}}`

Reply:
yep, I known `{{.SwaggerJSON}}` but that cause `variable doc(line 78)` useless.

Reply:
got it, you means it will cause `doc defined but not used` error,  right???
when i build my project this error not occurred, i will check it carefully

Reply:
no, I mean  `variable doc(line 78)` now don't be used. Because have using `{{.SwaggerJSON}}`instead of `variable doc(line 78)`.